newIDE/app/src/EventsSheet/ParameterRenderingService.js

```diff
@@ -103,7 +103,7 @@ const components = {
   atlasResource: AtlasResourceField,
   spineResource: SpineResourceField,
   color: ColorExpressionField,
-  police: DefaultField, //TODO
+  police: DefaultField, // TODO: Remove since fontResource is used now instead.
```

Comment:
`removed`

Reply:
Actually, it's a To remove 😅 
I'm not sure if I can do it right now. The only doubt I have is if an extension is using a "police" field somehow?